collective/deletepermission/testing.py

```diff
@@ -6,9 +6,13 @@
 from plone.app.testing import PLONE_FIXTURE
 from plone.app.testing import PloneSandboxLayer
 from plone.app.testing import setRoles, TEST_USER_ID, TEST_USER_NAME, login
+from pkg_resources import get_distribution
 import collective.deletepermission.tests.builders
 
 
+IS_PLONE_5_OR_GREATER = get_distribution('Plone').version >= '5'
```

Comment:
In testing, I'd rather use `ftw.testing.IS_PLONE_5`.

Reply:
We don't have `ftw.testing` installed. Should I install it only for this "constant"?